Shared/Samples/Style geometry types with symbols/StyleGeometryTypesWithSymbolsView.Views.swift

```diff
@@ -0,0 +1,245 @@
+            .onChange(of: selectedSize) { symbol.size = CGFloat($0) }
+            .onAppear {
+                selectedStyle = symbol.style
+                selectedColor = Color(uiColor: symbol.color)
+                selectedSize = Int(symbol.size)
+            }
+    }
+}
+
+/// Controls for editing a simple line symbol's properties.
+private struct SimpleLineSymbolEditor: View {
+    /// The simple line symbol to edit.
+    let symbol: SimpleLineSymbol
+    
+    /// The symbol's style selected by the picker.
+    @State private var selectedStyle: SimpleLineSymbol.Style = .noLine
+    
+    /// The symbol's color selected by the picker.
+    @State private var selectedColor: Color = .clear
+    
+    /// The symbol's width selected by the stepper.
+    @State private var selectedWidth: Int = .zero
+    
+    var body: some View {
+        EnumerationPicker("Style", selection: $selectedStyle)
+            .onChange(of: selectedStyle) { symbol.style = $0 }
+        
+        ColorPicker("Color", selection: $selectedColor)
+            .onChange(of: selectedColor) { symbol.color = UIColor($0) }
+        
+        Stepper("Width: \(selectedWidth)", value: $selectedWidth, in: 1...10)
+            .onChange(of: selectedWidth) { symbol.width = CGFloat($0) }
+            .onAppear {
+                selectedStyle = symbol.style
+                selectedColor = Color(uiColor: symbol.color)
+                selectedWidth = Int(symbol.width)
+            }
+    }
+}
+
+/// Controls for editing a simple fill symbol's properties.
+private struct SimpleFillSymbolEditor: View {
+    /// The simple fill symbol to edit.
+    let symbol: SimpleFillSymbol
+    
+    /// The symbol's style selected by the picker.
+    @State private var selectedStyle: SimpleFillSymbol.Style = .noFill
+    
+    /// The symbol's color selected by the picker.
+    @State private var selectedColor: Color = .clear
+    
+    var body: some View {
+        EnumerationPicker("Style", selection: $selectedStyle)
+            .onChange(of: selectedStyle) { symbol.style = $0 }
+        
+        ColorPicker("Color", selection: $selectedColor)
+            .onChange(of: selectedColor) { symbol.color = UIColor($0) }
+        
+        Section("Outline") {
+            SimpleLineSymbolEditor(symbol: symbol.outline as! SimpleLineSymbol)
+        }
+        .onAppear {
+            selectedStyle = symbol.style
+            selectedColor = Color(uiColor: symbol.color)
+        }
+    }
+}
+
+/// A picker for selecting a value from an enumeration's cases.
+private struct EnumerationPicker<T: LabeledEnumeration>: View {
+    /// The title of the picker.
+    private let title: String
+    
+    /// A binding to a property that determines the currently-selected style.
+    @Binding private var selection: T
+    
+    init(_ title: String, selection: Binding<T>) {
+        self.title = title
+        self._selection = selection
+    }
```

Comment:
Did you handwrite the initializer to omit the first argument label to mimic SwiftUI symbol names? If that's the reasoning, OK to have.

Reply:
Yes. I don't feel strong about it, though, so if you think it is unnecessary, I can remove it.